src/common/components/clusterWizard/ReviewValidations.tsx

```diff
@@ -64,31 +67,37 @@ const FailingValidation: FailingValidationsFC = ({
   wizardStepNames,
   wizardStepsValidationsMap,
 }) => {
-  const issue = `${
-    HOST_VALIDATION_LABELS[validation.id] ||
-    CLUSTER_VALIDATION_LABELS[validation.id] ||
-    validation.id
-  } check failed. `;
+  const { t } = useTranslation();
+
+  const issue = t('ai:{{check_failed}} check failed', {
+    check_failed:
+      HOST_VALIDATION_LABELS[validation.id] ||
```

Comment:
The texts on `HOST_VALIDATION_LABELS` are missing translations.

eg
```
export const HOST_VALIDATION_LABELS: { [key in HostValidationId]: string } = {
  'odf-requirements-satisfied': 'ODF requirements',
  'disk-encryption-requirements-satisfied': 'Disk encryption requirements',
```

Reply:
can be done in followup, in another PR